app/src/features/mappings/TableStep.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useEffect, useState } from "react";
+
+import { Icon } from "@blueprintjs/core";
+import { IconNames } from "@blueprintjs/icons";
+import {
+  Container,
+  Grid,
+  makeStyles,
+  Button,
+  Typography,
+} from "@material-ui/core";
+import AddIcon from "@material-ui/icons/AddCircleOutline";
+import clsx from "clsx";
+import { useTranslation } from "react-i18next";
+
+import Select from "common/Select/Select";
+import {
+  Resource,
+  useListOwnersQuery,
+} from "services/api/generated/api.generated";
+
+const useStyles = makeStyles((theme) => ({
+  button: {
+    textTransform: "none",
+  },
+  icon: {
+    paddingRight: theme.spacing(1),
+    fill: theme.palette.text.disabled,
+  },
+  iconSelected: {
+    fill: theme.palette.secondary.main,
+  },
+  inputSelected: {
+    fontWeight: 500,
+  },
+}));
+
+type TableStepProps = {
+  mapping: Partial<Resource>;
+  onChange?: (mapping: Partial<Resource>) => void;
+};
+
+const TableStep = ({ onChange, mapping }: TableStepProps): JSX.Element => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+  const { data: owners } = useListOwnersQuery({});
```

Comment:
Here you need to filter the owners by credential. The filter is on my PR though...